scripts/create-user.sh

```diff
@@ -227,7 +196,12 @@ get_subject() {
 
 # Expand variables from message body, escaping new lines and quotes
 get_body() {
-  eval "echo \"${MESSAGE}\"" | \
+  erb email="${EMAIL}" org="${ORG:-}" invite_url="${INVITE_URL}" "${SCRIPT_DIR}/templates/create_user_plain.erb" | \
```

Comment:
We should add `erb` to the `Gemfile` now that we're using it, and probably run it with `bundle exec` here.

Reply:
erb is part of the standard library so we don't need to install it

Reply:
Thanks for clarifying that... I hadn't realised this. TIL.